README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
The project is not deployed. If it was the case, the url would be https://fleuronvilik.github.io/reviewing-a-pull-requestl/.